Podfile

```diff
@@ -47,7 +47,7 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 4.58.2-beta.1'
+  pod 'WordPressKit', '~> 4.58', '>= 4.58.2'
```

Comment:
I'm still not convinced that this `>= …` additional constraint is useful, because what matters is that the lockfile has been updated to lock onto the latest version (with those fixes from `4.58.2`), which it has. So I think having the `Podfile` just having `~> 4.58`, without the additional `>= 4.58.2`, would be enough, as long as you ensured to run `pod update` and have the lockfile (which is the source of truth for which versions of each lib has been locked, after all) point to a version which has all the fixes you want (and potentially more).

🤷